assets/responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -23,6 +23,10 @@ RUN pip install 'azureml-dataset-runtime=={{latest-pypi-version}}' \
                 'azureml-telemetry=={{latest-pypi-version}}' \
                 'Werkzeug==2.2.3'
 
+# azureml-dataset-runtime[fuse] upper bound for pyarrow is 11.0.0
+# so we install pyarrow in extra step to avoid conflict
+RUN pip install 'pyarrow==14.0.1'
```

Comment:
Why are we == pinning to 14.0.1? Looks like the conda file it was >=.

Reply:
https://msdata.visualstudio.com/Vienna/_git/AzureMlCli/pullrequest/1221644?_a=files

This is what azureml is doing which is why i pinned to that exact version